src/dashboard/frontend/src/index.css

```
@@ -2,6 +2,91 @@
+  font-display: swap;
+  font-weight: 300 700;
+  src: url('/fonts/SpaceGrotesk-Variable.woff2') format('woff2-variations'), url('/fonts/SpaceGrotesk-Variable.woff2') format('woff2');
+}
+
+/* ─── Theme font tokens (PAN-3410 D-13) — :root carries Ledger's exact values;
+     [data-theme="broadsheet"] overrides them. Components never branch on
+     theme directly, only on these variables. ─── */
+:root {
+  --font-sans: "DM Sans", system-ui, sans-serif;
+  --font-display: "Space Grotesk", system-ui, sans-serif;
+  --font-mono: "SF Mono", "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.1)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)